### PR TITLE
feat: create ui for add user form

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image
image

### Things to Talk About
Plz check by commits!

### CODE REVIEW

File: apps/web/src/services/administration/components/UserManagementAddUser.vue [hunk 1]
```diff
@@ -118,22 +167,19 @@ const fetchListUsers = async () => {
...
-        });
-        formState.searchText = '';
-    } else {
-        validationState.userIdInvalid = true;
-        validationState.userIdInvalidText = i18n.t('IDENTITY.USER.FORM.USER_ID_INVALID_WORKSPACE', { userId });
-    }
+    validationState.userIdInvalid = true;
+    validationState.userIdInvalidText = i18n.t('IAM.USER.FORM.USER_ID_INVALID_WORKSPACE', { userId });
+    await hideMenu();
+};
```
Comment (piggggggggy) [inline apps/web/src/services/administration/components/UserManagementAddUser.vue:176]: Question: I wonder why don't use response

**Review by piggggggggy — APPROVED** [review]
LGTM~

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/administration/components/UserManagementAddUser.vue [hunk 2]
```diff
@@ -154,32 +201,51 @@ watch(() => state.menuVisible, async (menuVisible) => {
...
-                                    ref="contextMenuRef"
-                                    :loading="state.loading"
-                                    :menu="state.menuItems"
-                                    :selected="state.selectedItems"
-                                    multi-selectable
-                                    class="user-context-menu"
-                                    @select="handleSelectMenuItem"
-                    />
+                    <div v-if="userPageState.isAdminMode">
+                        <p-select-dropdown :menu="contextMenuItems"
```
Comment (sulmoJ) [inline apps/web/src/services/administration/components/UserManagementAddUser.vue:224]: suggestion: How about authTypeMenuItem?

**Review by sulmoJ — APPROVED** [review]
LGTM~~

**Pull request merged**